lib/mix/lib/mix/task.ex

```diff
@@ -347,29 +358,46 @@ defmodule Mix.Task do
   task is invalid. See `get!/1` for more information.
   """
   @spec run(task_name, [any]) :: any
-  def run(task, args \\ [])
+  def run(task, args \\ []) do
+    do_run(task, args)
+  end
+
+  @doc """
+  Similar to run/2 but it runs the task in the specified list of children apps,
+  in the context of umbrella proyects.
+
+  However, if the task is not recursive (whose purpose is to be run in children
+  applications) it runs at the project root level (a warning message is shown
+  in this case).
+  """
+  @spec run_in_apps(task_name, [atom], [any]) :: any
```

Comment:
Do we need to add `@doc since: ..`? Which version?

Reply:
Yes. 1.14.0!